lib/canvas_link_migrator/resource_map_service.rb

```diff
@@ -101,6 +101,13 @@ def convert_migration_id(type, migration_id)
       # the /discusson_topic url scheme is used for annnouncments as well.
       # we'll check both here
       convert_announcement_migration_id(migration_id) if type == "discussion_topics"
+      if LinkParser::KNOWN_REFERENCE_TYPES.include? type
```

Comment:
I think this is duplicated from line 97?

Reply:
There was a merge conflict, and I think I messed it up.  I'll fix it.

Reply:
This should be fixed now.